docs/meetings/exeter-jun-2019.md

```diff
@@ -0,0 +1,254 @@
+# Exeter Meeting Notes
+
+June 24-28
+Present HO, DM, MS, OS, SB, BK
```

Comment:
TP

Reply:
Very sorry @wxtim - you are in fact exceedingly memorable!

Reply:
Noted.